ddtrace/appsec/_asm_request_context.py

```diff
@@ -70,8 +70,13 @@ def __init__(self, span: Optional[Span] = None):
         if context_span is None:
             info = add_context_log(log, "appsec.asm_context.warning::ASM_Environment::no_span")
             log.warning(info)
-            context_span = tracer.trace("asm.context")
+            context_span = tracer.trace("sdk.request")
```

Comment:
Why this change? What’s the difference between sdk.request and asm.context?

Reply:
this piece of code is just a failsafe that should never be used. But in case we need it, it means this function was used outside of its regular usage, that's why I changed it to sdk.request (manual instrumentation).